### DISCUSSION TITLE
Java support

### DISCUSSION THREAD

**Discussion by MalfiRG** [Codium-ai/codiumai-jetbrains-release#26] [Ideas]
Hi! Is it possible to implement CodiumAI in a Java project? If not, do you plan to implement Java support in your future releases?

**Comment by almog-lv** [comment]
Hello, thank you for your interest in CodiumAI. We are planning to support more languages, Java is not yet supported but is definitely on our short list of next candidates. Stay tuned!

**Reply by MalfiRG** [reply]
Great! I can't wait for the Java support implementation; the project has big potential!
